sfm/ui/views.py

```diff
@@ -142,6 +142,18 @@ def get_context_data(self, **kwargs):
             seed_warning_message = "At least 1 active seed must be added before harvesting can be turned on."
         context["seed_error_message"] = seed_error_message
         context["seed_warning_message"] = seed_warning_message
+
+        # Adding credential active message
+        context["credential_used_col_list"] = None
+        credential_warning_message = None
+        if self.object.harvest_type in [Collection.TWITTER_FILTER, Collection.TWITTER_SAMPLE]:
+            context["credential_used_col_list"] = Collection.objects.filter(credential=self.object.credential.pk,
```

Comment:
This list might have 0 or 1 entries, correct?  If so, then don't pass a list, pass None or the single entry.  This will remove the need for the list iteration in the template.